lib/src/prices/get_prices_parameters.dart

```diff
@@ -1,3 +1,5 @@
+import 'package:openfoodfacts/src/prices/price_kind.dart';
```

Comment:
```suggestion
import 'price_kind.dart';
```
Please put direct path here.